src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -70,11 +87,18 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
     //   used in key specification without a key length
     //
     // Perhaps we'll use a more optimized data type for UUID's bytes in the future, as a FK, instead auto-inc cols
+    //
+    // The value `${slick.uniqueSchema}` may used in the url, in combination with `slick.createSchema = true`, to
```

Comment:
"may be"

Also, I think I now get the "why" in my previous comment, although it'd be good to see it in plain english to make sure I understand